code/modules/clothing/under/_under.dm

```diff
@@ -104,6 +104,7 @@
 		//make the sensor mode favor higher levels, except coords.
 		sensor_mode = pick(SENSOR_OFF, SENSOR_LIVING, SENSOR_LIVING, SENSOR_VITALS, SENSOR_VITALS, SENSOR_VITALS, SENSOR_COORDS, SENSOR_COORDS)
 	sensor_mode_intended = sensor_mode
+	register_context()
```

Comment:
is this runtiming because it's under New instead of Initialize?

Reply:
oh wait you changed it from initialize... that's weird that it would still be doing that

Reply:
It was already new, i changed to initialize then back, how the runtime happens doesn't make sense.

Reply:
i mean yeah you tried to as initialize and it didn't work, which is weird .....